### PR TITLE
Fahimeh/promote code clean up

### PR DESCRIPTION
**Pull request by FahimehMirhaj**
The sequencing requirement plugin handles the changes that was removed from promote code.
Please review the changes.

### CODE REVIEW

File: src/dev/resources/connect.txt [hunk 1]
```diff
@@ -7,4 +7,10 @@ lims.pword1=
 lims.user2=
 lims.pword2=
 
-# TODO - get credentials
```
Comment (DavidStreid) [inline src/dev/resources/connect.txt:10]: Can we keep `user`/`pword` commented out?

**Review by DavidStreid — COMMENTED** [review]
Looks good! Can we hide the credentials?

File: src/dev/resources/connect.txt (hunk 1 above)
Comment (DavidStreid) [reply src/dev/resources/connect.txt:10]: Oh sorry, not commented out - deleted? Like this - 
```
lims.host=
lims.port=
lims.guid=
lims.user1=
lims.pword1=
lims.user2=
lims.pword2=
```

Comment (DavidStreid) [reply src/dev/resources/connect.txt]: looks good, thanks!

**Review by DavidStreid — APPROVED** [review]
Yes, cleaner code :D

**Review by apatruno618 — APPROVED** [review]

**Review by dmcmanam — APPROVED** [review]

**Pull request merged**